ripc-protocol-tcp/src/main/java/io/ripc/protocol/tcp/TcpHandler.java

```diff
@@ -0,0 +1,8 @@
+package io.ripc.protocol.tcp;
+
+import org.reactivestreams.Publisher;
+
+public interface TcpHandler<R, W> {
```

Comment:
> The reason I added Tcp prefix was that Handler by itself is confusing when someone starts creating two servers with different transports in the same code.

Good point.

Maybe this should be applied across-the-board so that `TcpConnection` doesn't interfere with `WebSocketConnection` et al?